# File: msg/DetectionWithTrackID.msg

# Custom message with Detection2D and track_id
vision_msgs/Detection2D detection
int64 track_id

